README.md

```diff
@@ -87,6 +88,7 @@ augroup autoformat_settings
   autocmd FileType java AutoFormatBuffer google-java-format
   autocmd FileType python AutoFormatBuffer yapf
   " Alternative: autocmd FileType python AutoFormatBuffer autopep8
+  autocmd FileType rust AutoFormatBuffer rust
```

Comment:
Shouldn't this be `AutoFormatBuffer rustfmt`? The argument is a formatter name.

Reply:
Done.